yt_dlp/extractor/youtube.py

```diff
@@ -2402,19 +2415,8 @@ def _extract_player_responses(self, clients, video_id, webpage, master_ytcfg, pl
         while clients:
             client = clients.pop()
             player_ytcfg = master_ytcfg if client == 'web' else {}
-            r'''  # This doesnt have thirdParty set - Remove?
             if 'configs' not in self._configuration_arg('player_skip'):
                 player_ytcfg = self._extract_player_ytcfg(client, video_id) or player_ytcfg
```

Comment:
If we are using this ytcfg, we should manually set `thirdParty`, right?

Reply:
does the embedded client need this?

Reply:
huh, you are right. It doesn't seem to help in any of the cases. Not sure why I thought it was needed